src/scripts.js

```diff
@@ -35,35 +38,31 @@ pendingTripsButton.addEventListener("click", (event) => {
 });
 
 //Functions//
-const getRandomIndex = (array) => {
-  return Math.floor(Math.random() * array.length);
-};
-
 const loadData = () => {
   fetchAll()
     .then((data) => {
-      const id = 30;
+      const id = 35;
       const [travelersData, tripsData, destinationsDataObj] = data;
-      const destinationsData = destinationsDataObj.destinations;
+      destinationsArray = destinationsDataObj.destinations;
       const travelersRepository = new TravelersRepository(
         travelersData.travelers
       );
       const traveler11Data = travelersRepository.getSingleTraveler(id);
       const tripsRepository = new TripsRepository(tripsData.trips);
       const traveler11Trips = tripsRepository.getTrips(id);
       currentTraveler = new Traveler(traveler11Data, traveler11Trips);
-      // console.log(destinationsData);
-      return { currentTraveler, destinationsData };
-    })
-    .then(({ currentTraveler, destinationsData }) => {
-      startApplication(currentTraveler, destinationsData);
+      return { currentTraveler, destinationsArray };
     })
-    .catch((error) => console.log(`There has been an error! ${error}`));
+    .then(({ currentTraveler, destinationsArray }) => {
+      startApplication(currentTraveler, destinationsArray);
+    });
+  // .catch((error) => console.log(`There has been an error! ${error}`));
 };
 
-const startApplication = (user, destinationsData) => {
+const startApplication = (user, destinationsArray) => {
```

Comment:
Great use of this helper function to call all these functions for the page load.